.editorconfig

```diff
@@ -1,7 +1,7 @@
 root = true
 
 [*]
-indent_style = space
+indent_style = tab
```

Comment:
And then I think we want to have `indent_style = space` for `*.xml` below. Also the whole `*.java` block can be removed.

(Or set this `indent_style = tab` in the `*.java` block instead of the global one.)

Reply:
fixed